src/css/controls/flags/live.less

```diff
@@ -37,3 +31,41 @@
         display: none;
     }
 }
+
+.jw-text-live {
+    .rect(auto, @icon-size);
+    align-items: center;
+    border-radius: 1px;
+    color: @inactive-color;
+    display: flex;
+    font-size: 12px;
+    font-weight: bold;
+    margin-right: 10px;
+    padding: 0 1ch;
+    text-rendering: geometricPrecision;
```

Comment:
There's no support for this prop in IE. Does it look fine without it there?

Reply:
Having this property is to improve consistency in Chrome on MacOS. Firefox treats this value as `optimizeLegibility` (which doesn't have any effect on Arial), Safari doesn't do anything with the property, and IE/Edge use DirectWrite and render the text properly.